src/main/resources/config/liquibase/changelog/20211006150000_add_person_name.xml

```diff
@@ -21,12 +21,12 @@
     -->
     <changeSet id="20211006150000-1" author="alexey@thehyve.nl">
         <addColumn tableName="subject">
-            <column name="person_name" type="varchar(255)"/>
+            <column name="person_name" type="varchar(255)" defaultValue="John Doe"/>
         </addColumn>
     </changeSet>
     <changeSet id="20211006150000-2" author="alexey@thehyve.nl">
         <addColumn tableName="subject_aud">
-            <column name="person_name" type="varchar(255)"/>
+            <column name="person_name" type="varchar(255)" defaultValue="John Doe"/>
```

Comment:
Similar to above
> No changes should be made to existing Liquibase changeSets, because it leads to undefined state of the database. If you want to add a default value (I think we shouldn't, it should be null), a new changeset should be created.

Reply:
added a changeset with some testvalues in #713

Reply:
This default value should still be removed. It also affects production databases.